src/components/registration/steps.jsx

```diff
@@ -324,13 +324,26 @@ module.exports = {
         },
         getYearOptions: function () {
             return Array.apply(null, Array(100)).map(function (v, id) {
-                var year = new Date().getFullYear() - id;
+                var year = new Date().getFullYear() - (id + 13);
```

Comment:
Since this is going to eliminate the last 12 years from the options list, I assume getYearOptions is only used for Teachers.